VVResonances/scripts/vvMake2DTemplateWithKernels.py

```diff
@@ -80,26 +94,52 @@ def conditional(hist):
         for j in range(1,hist.GetNbinsX()+1):
             hist.SetBinContent(j,i,hist.GetBinContent(j,i)/integral)
             
+
 def smoothTail(hist):
```

Comment:
since we are not using any smoothing, what about removing it completely even from the central repo. perhaps keep your local copy in case at some point we want to study this a bit more.